website/layouts/partials/versioning/banner.html

```diff
@@ -0,0 +1,13 @@
+{{- $currentVersion := index (split .Dir "/") 0 }}
+{{- $latestVersion := "tip" }}
+{{- $latestUrl := replace .URL $currentVersion $latestVersion }}
+
+{{- if ne $currentVersion $latestVersion }}
+<div class="alert alert-warning" role="alert">
+    <strong>This document is for an earlier version of Thanos.
+        {{ if (fileExists $latestUrl) -}}
+             Click <a href="{{ $latestUrl }}">here </a> to see the latest version of this page.
```

Comment:
Can we include what actually past version we are looking on

Reply:
Done!